src/TimePicker/_TimePickerItem.js

```diff
@@ -253,6 +253,10 @@ TimePickerItem.propTypes = {
     inputProps: PropTypes.object,
     isValid: PropTypes.bool,
     length: PropTypes.number,
+    localizedText: CustomPropTypes.i18n({
+        meridiemAM: PropTypes.string,
+        meridiemPM: PropTypes.string
```

Comment:
Same question as other pr on Time, could `TimePicker` pass these props to `_TimePickerItem` so that consumers only have to provide the localized text once?

Reply:
@jbadan totally agree! since they were originally coming in as global variables, this is the way @greg-a-smith and I had talked about solving it for the meantime before we end up changing time and timepicker altogether with moment.js(or similar). @greg-a-smith do you think its possible to pass props only once as it is right now?

Reply:
Yes.  See my comments in #495.  The same pattern will occur here.  The `TimePicker` component will use the `i18n` custom prop type validator and then `TimePickerItem` can just use the `object` prop type validator.

Reply:
Yes.  See my comments in #495.  The same pattern will occur here.  The `TimePicker` component will use the `i18n` custom prop type validator and then `TimePickerItem` can just use the `object` prop type validator.